pkg/network/listener.go

```diff
@@ -204,8 +206,10 @@ func (l *listener) Stop() error {
 	switch l.network {
 	case "udp":
 		err = l.packetConn.SetDeadline(time.Now())
-	default:
-		err = l.rawl.SetDeadline(time.Now())
+	case "unix":
+		err = l.rawl.(*net.UnixListener).SetDeadline(time.Now())
```

Comment:
The function of [GetInheritListeners](https://github.com/mosn/mosn/blob/master/pkg/server/handler.go#L886) needs to be adapted.